runtime/cudaq/platform/default/rest/helpers/iqm/IQMServerHelper.cpp

```diff
@@ -259,9 +259,12 @@ IQMServerHelper::generateRequestHeader() const {
 
 void IQMServerHelper::updatePassPipeline(
     const std::filesystem::path &platformPath, std::string &passPipeline) {
+  // Note: the leading and trailing single quotes are needed in case there are
+  // spaces in the file.
```

Comment:
"spaces in the filename"

Reply:
> "spaces in the filename"

Updated in https://github.com/NVIDIA/cuda-quantum/pull/1081